gradle/compile.gradle

```diff
@@ -36,9 +37,15 @@ tasks.withType(JavaCompile) {
     // while compiling.
     options.compilerArgs = ['-Xlint:-options', '-Xlint:-path']
     options.incremental = true
+    inputs.property('javaVendor') {
+        java7Home ? 'Oracle Corporation' : javaVendor
```

Comment:
I think you actually don't know. Because while on CI, we know what is installed, for local builds it's another game. I would use the probe to make sure.